Remove Gitter links

jsumners opened the pull request:
This PR removes the Gitter links from the readme. People are using it as a "your our only hope @mcollina" channel because 9 times out of 10 he is the only one answering anything. I _occasionally_ answer some things there, but not nearly enough. The community is not helping itself as desired through this avenue. Thus, I am proposing this PR and subsequently closing down the Gitter channel.

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

mcollina commented on README.md:
```diff
@@ -193,7 +193,6 @@ matters to you.
 
 ## Support
 - [Fastify help](https://github.com/fastify/help)
```
Can you add some text before this link? A list of only 1 element is not nice.

jsumners replied:
Updated.

jsumners commented:
> A few other resources would need to be updated:
> 
> https://github.com/fastify/.github/blob/master/.github/ISSUE_TEMPLATE/bug.md
> 
> https://github.com/fastify/.github/blob/master/.github/ISSUE_TEMPLATE/feature.md
> 
> https://github.com/fastify/.github/blob/master/.github/ISSUE_TEMPLATE/question.md
> 
> https://github.com/fastify/.github/blob/master/.github/ISSUE_TEMPLATE/regression.md
> 
> https://github.com/fastify/fastify/blob/master/.github/ISSUE_TEMPLATE.md

Can I get maintainership access to that repo?

mcollina commented:
@jsumners this is in this repo to fix: https://github.com/fastify/fastify/blob/master/.github/ISSUE_TEMPLATE.md

mcollina commented:
@jsumners you should have access to that repo.

jsumners commented:
@mcollina oops. I was thinking it was the `fastify/.github` repository you were telling me about. Anyway, I have updated the files there (see https://github.com/fastify/.github/pull/2) and removed them here. After this PR is merged, the global versions in `fastify/.github` should be used.
